src/components/KMenu/KMenuItem.vue

```diff
@@ -29,17 +29,17 @@
         class="span-icon-container"
       >
         <KIcon
-          color="var(--grey-400)"
+          :color="`var(--grey-400, var(--kui-color-text-neutral-weak, ${KUI_COLOR_TEXT_NEUTRAL_WEAK}))`"
           :icon="isOpen ? 'chevronUp' : 'chevronDown'"
-          size="16"
+          :size="KUI_ICON_SIZE_30"
         />
       </span>
     </KButton>
 
     <div
       v-if="expandable"
       class="menu-content"
-      :class="isOpen ? 'd-flex' : 'd-none'"
+      :class="isOpen ? 'is-open' : 'is-close'"
```

Comment:
```suggestion
      :class="isOpen ? 'is-open' : 'is-closed'"
```
nit (don't forget to update corresponding selector in style section please)

Reply:
updated both